lib/dl_formula/dl_formula/definitions/base.py

```diff
@@ -677,6 +688,14 @@ def for_dialect(
     ) -> _SINGLE_NODE_TRANS_TV:
         return cls(dialects=dialects)
 
+    def for_another_dialect(  # type: ignore  # 2024-02-20 # TODO: Signature of "for_another_dialect" incompatible with supertype "MultiVariantTranslation"  [override]
```

Comment:
let's not add new type ignores if it's not absolutely necessary
in this case, there is an interface missing in the base class (check `for_dialect` method for example)